tests/gdb-tests/tests.sh

```diff
@@ -67,7 +74,11 @@ tests_passed_or_skipped=0
 tests_failed=0
 
 for test_case in ${TESTS_LIST}; do
-    gdb_args=(-ex 'py import coverage;coverage.process_startup()' --command $GDB_INIT_PATH --command pytests_launcher.py)
+    if [ ${RUN_CODECOV} -eq 0 ]; then
+        gdb_args=(--command $GDB_INIT_PATH --command pytests_launcher.py)
+    else
+        gdb_args=(-ex 'py import coverage;coverage.process_startup()' --command $GDB_INIT_PATH --command pytests_launcher.py)
+    fi
```

Comment:
In the interest of reducing the amount of code we repeat, we can prepend arguments here instead (this is untested):
```bash
gdb_args=(--command $GDB_INIT_PATH --command pytests_launcher.py)
if [ ${RUN_CODECOV} -ne 0 ]; then
        gdb_args=(-ex 'py import coverage;coverage.process_startup()' "${gdb_args[@]}")
fi
```